### PR TITLE
Drop smart_proxy_dynflow Jenkins test changes

### PR DESCRIPTION
**Pull request by archanaserver**
For refernce: https://github.com/theforeman/smart_proxy_dynflow/pull/123#issuecomment-1646601180

We have GitHub Action to run the tests against different Ruby versions.

### CODE REVIEW

**Comment by archanaserver** [comment]
@ehelms does this looks good?

File: Gemfile [hunk 1]
```diff
@@ -16,7 +16,6 @@ end
 
 group :test do
   gem 'minitest'
-  gem 'minitest-reporters'
```
Comment (ehelms) [inline Gemfile:19]: I think keeping this is good, as these reporters are useful for more than just Jenkins. @ekohl ?

Comment (archanaserver) [reply Gemfile:19]: Should I keep it now @ehelms @ekohl? I don't see any other configuration for it to use, other than Jenkins in this project.

Comment (ekohl) [reply Gemfile:19]: It slightly changes the output and gives a progress bar but I'm not sure I ever found that very useful. I'm fine with either way.

**Review by ekohl — APPROVED** [review]
@adamruzicka any preference on minitest-reporters?

**Comment by adamruzicka** [comment]
I wouldn't mind if they went away.

**Pull request merged**